samples/bookinfo/routing-rules/route-rule-all-v1.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
I would suggest defining the destination rules with the subsets first.. and then define the virtual services. May be even in two separate files (one for all subset definitions, and another for all virtual services)

Reply:
and nit.. file is virtual-service-..